config/basal_fitting_checks.csv

```diff
@@ -0,0 +1,28 @@
+BMI,CHO,TDD,MIN_OUTPUT,MAX_OUTPUT
+12,0,0,-500,0
+12,250,0,-500,0
+12,500,0,-500,0
+25,0,0,-500,0
+25,250,0,-500,0
+25,500,0,-500,0
+45,0,0,-500,0
+45,250,0,-500,0
+45,500,0,-500,0

```

Comment:
Are we OK with having these combinations give negative results? What is the intention of requiring them to be negative?

Reply:
Great question. We are Trying to solve for the TDD = 0 then basal should be =0. We realized that requiring the basal to be 0 (we had basal min = 0 and max= 0) resulting in no models passing this check. However, allowing the results to be negative allowed the models to converge and a negative basal results are not realistic, so the end user will quickly realize that results are not defined. Does that make sense?

Reply:
Makes sense to me!